extensionsGallery.json

```diff
@@ -3765,6 +3765,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "77",
+					"extensionName": "pastelin",
+					"displayName": "Pastelin",
+					"shortDescription": "Pastelin - Azure Data Studio Theme (Dark)",
+					"publisher": {
+						"displayName": "Tanyo Georgiev",
+						"publisherId": "tanyogeorgiev",
+						"publisherName": "tanyogeorgiev"
+					},
+					"versions": [
+						{
+							"version": "1.0.1",
+							"lastUpdated": "10/26/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/tanyogeorgiev/Pastelin/releases/tag/1.0.1"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://github.com/tanyogeorgiev/Pastelin/blob/main/assets/logo.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://raw.githubusercontent.com/tanyogeorgiev/Pastelin/main/README.md"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Code.Manifest",
+									"source": "https://raw.githubusercontent.com/tanyogeorgiev/Pastelin/main/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://raw.githubusercontent.com/tanyogeorgiev/Pastelin/main/LICENSE"
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Services.Links.Source",
+									"value": "https://github.com/tanyogeorgiev/Pastelin"
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": "*"
+								}
+							]
+						}
+					],
+					"statistics": [],
+					"flags": "preview"
```

Comment:
your extension's version is 1.xx already, do you still want to mark it as preview?

Reply:
I'm OK to remove "preview" mark. Thanks...